trino/etc/secrets/generate_jwt.py

```diff
@@ -0,0 +1,17 @@
+import jwt
+from datetime import datetime, timedelta
+
+# Define the payload for the JWT token
+payload = {
+    'sub': 'test',
+    'exp': datetime.utcnow() + timedelta(days=36500)  # Expiration time (e.g., 1 day)
+}
+
+# Load the RSA private key
+with open('private_key.pem', 'rb') as f:
+    private_key = f.read()
+
+# Generate JWT token
+token = jwt.encode(payload, private_key, algorithm='RS256')
+
+print(token) 
```

Comment:
When this should be used? Maybe mention it in the README or test docs. Can this be a Go program?

Reply:
@nineinchnick - This is the code I used to generate the token. I can change to Go and I will add README.

Reply:
@nineinchnick - 
* Added README 
* Removed not used secret files
* Added Go based JWT token generator
* Increase expiry time for certificate/token